### ISSUE TITLE
Display length of show/movie in hour minute format

### ISSUE THREAD

**Issue by d8f6** [jellyfin/jellyfin-androidtv#2806]
### Describe the feature you'd like

Currently the length of shows/movies show the length of the media in terms of minutes only. For example: a movie that is an hour and forty five minutes long is displayed as: "105m". What I would like instead is for the time remaining to display as "1h 45m". This would be consistent with the web player/desktop player. Seems like a relatively simple fix. Okay, thanks.

**Comment by jacobthemagicalpanda** [comment]
I would love to see this or at least have it as an option. I am bad at math in my head and can't just glance and know how long something is like that

**Comment by jim383** [comment]
I think this should be closed as Fixed by #4253

**Comment by nielsvanvelzen** [comment]
Yup not sure how the issue wasn't auto-closed. Closing it now as fixed (for 0.19!)
